packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx

```diff
@@ -296,6 +315,62 @@ describe('Vertical stacked bar chart - Subcomponent Legends', () => {
       expect(legendsAfterClickEvent[3]).toHaveAttribute('aria-selected', 'false');
     },
   );
+
+  testWithWait(
+    'Should call the handler on mouse leave from bar legend',
+    VerticalStackedBarChart,
+    { data: simplePoints, calloutProps: { doNotLayer: true } },
+    container => {
+      // eslint-disable-next-line
+      const handleMouseLeave = jest.spyOn(VerticalStackedBarChartBase.prototype as any, '_onLegendLeave');
+      const legends = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'button');
+      fireEvent.mouseLeave(legends[0]);
+      // Assert
+      expect(handleMouseLeave).toHaveBeenCalled();
+    },
+  );
+
+  testWithWait(
+    'Should call the handler on mouse leave from line legend',
+    VerticalStackedBarChart,
+    { data: simplePointsWithLine, calloutProps: { doNotLayer: true } },
+    container => {
+      // eslint-disable-next-line
+      const handleMouseLeave = jest.spyOn(VerticalStackedBarChartBase.prototype as any, '_onLegendLeave');
+      const legends = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'button');
+      fireEvent.mouseLeave(legends[0]);
```

Comment:
I dont see any difference between this test and the previous test

Reply:
removed duplicate test case